packages/frontend/src/lib/pnl.ts

```diff
@@ -0,0 +1,43 @@
+import BigNumber from 'bignumber.js'
```

Comment:
would it makes sense to add this to utils instead of having a separate lib folder just for consistency or do you see lib as distinct from utils?

Reply:
Yes, I think utils is for more general things and a lib folder is good to have to keep calculations that are directly associated with each hook.